modules/ibi-extra-manifests-configmap.adoc

```diff
@@ -0,0 +1,140 @@
+
+The following example adds an single-root I/O virtualization (SR-IOV) network to the deployment.
+
+.Prerequisites
+
+* You preinstalled a host with {sno} using an image-based installation.
+* You logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `SriovNetworkNodePolicy` and `SriovNetwork` resources:
+
+.. Create a YAML file that defines the resources:
++
+.Example `sriov-extra-manifest.yaml` file
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: "example-sriov-node-policy"
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames: [ens1f0]
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  mtu: 1500
+  numVfs: 8
+  priority: 99
+  resourceName: example-sriov-node-policy
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: "example-sriov-network"
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: |-
+    {
+    }
+  linkState: auto
+  networkNamespace: sriov-namespace
+  resourceName: example-sriov-node-policy
+  spoofChk: "on"
+  trust: "off"
+----
+
+.. Create the `ConfigMap` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create configmap sr-iov-extra-manifest --from-file=sriov-extra-manifest.yaml -n openshift-config
+----
++
+.Example output
+[source,terminal]
+----
+configmap/sr-iov-extra-manifest created
+----
+
+. Reference the `ConfigMap` resource in the `spec.extraManifestsRefs` field of the `ImageClusterInstall` resource:
++
+[source,yaml]
+----
+#...
+  spec:
+    extraManifestsRefs:
+    - name: sr-iov-extra-manifest
+#...
+----
+
+== Creating a ConfigMap resource to add a CA bundle to the host
+
+You can use a `ConfigMap` resource to add a certificate authority (CA) bundle to the host to ensure trusted communications for cluster services. 
+
+After you create the `ConfigMap` resource, reference it in the `spec.caBundleRef` field of the `ImageClusterInstall` resource.
+
+.Prerequisites
+
+* You preinstalled a host with {sno} using an image-based installation.
+* You logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a CA bundle file such as the following file:
++
+.Example `example-ca.crt`
+[source,text]
+----
+-----BEGIN CERTIFICATE-----
+MIIDXTCCAkWgAwIBAgIJAKmjYKJbIyz3MA0GCSqGSIb3DQEBCwUAMEUxCzAJBgNV
+...Custom CA certificate bundle...
+4WPl0Qb27Sb1xZyAsy1ww6MYb98EovazUSfjYr2EVF6ThcAPu4/sMxUV7He2J6Jd
+cA8SMRwpUbz3LXY=
+-----END CERTIFICATE-----
+----
+
+. Create the ConfigMap object by running the following command: 
++
+[source,terminal]
+----
+$ oc create configmap custom-ca --from-file=example-ca.crt -n openshift-config
+----
++
+.Example output
+[source,terminal]
+----
+configmap/custom-ca created
+----
+
+. Reference the `ConfigMap` resource in the `spec.extraManifestsRefs` field of the `ImageClusterInstall` resource:
```

Comment:
You wrote it right https://github.com/openshift/openshift-docs/pull/79610/files#diff-044c4ecad9674319539b0bbd91a2715c7630137dfb44a10cc9128add50491fadR95 but here for some reason we have extraManifestsRefs ?

Reply:
Oops, copy-paste error. Thanks!